ui/src/screen/project/source.ts

```diff
@@ -284,14 +284,18 @@ export const fetchCommit = async (sha1: string): Promise<void> => {
     try {
       commitStore.success(await source.fetchCommit(project.urn, sha1));
     } catch (err: unknown) {
-      commitStore.error(error.fromUnknown(err));
-      error.show(
-        new error.Error({
-          code: error.Code.CommitFetchFailure,
-          message: "Could not fetch commit",
-          source: err,
-        })
-      );
+      const e = error.fromUnknown(err);
+      if (!e.message.match("object not found")) {
```

Comment:
Could you flip the `if/else` branches and get rid of the negation?

Reply:
51bf94ca894826c5b804aaf25b2310c2ee11c51d